app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -81,6 +102,28 @@ define(function (require, exports, module) {
       return ! account.has('profileImageUrl');
     },
 
+    _addLoadingSpinner: function ($wrapperClass) {
+      $wrapperClass.append('<span class="avatar-spinner"></span>');
+    },
+
+    // "Completes" the spinner, transitioning the semi-circle to a circle, and
+    // then removes the spinner element.
+    _completeLoadingSpinner: function ($wrapperClass) {
+      var deferred = p.defer();
+      var $spinner = $wrapperClass.find('.avatar-spinner');
+      $spinner.addClass('completed')
+        .on('transitionend', function (evt) {
```

Comment:
Does jQuery take care of the WebKit variant? See [the notes](http://caniuse.com/#search=transitionend) on caniuse.com. I imagine it does, but want to make sure.